### ISSUE TITLE
Extension won't respect <file> settings

### ISSUE THREAD

**Issue by pattisahusiwa** [wongjn/vscode-php-sniffer#19]
**Describe the bug**
There are different results between the extension and running `phpcs` directly. Please test the attachment file to see the bug. On that simple project, `phpcs` should inspect files under `src` dir only.

**To Reproduce**
1. Open the project file (`testbug.code-workspace`).
2. Open `test.php` under `test` directory
3. The extension will report some violations.

**Expected behavior**
No error reported.

**Environment**
 - OS: Arch Linux
 - VSCode version: 1.35.1
 - PHP CLI version: 7.3.6
-  squizlabs/PHP_CodeSniffer version: 3.4.0

**Extension settings**
```json
{
  "phpSniffer.executablesFolder": "",
  "phpSniffer.onTypeDelay": 250,
  "phpSniffer.run": "onSave",
  "phpSniffer.snippetExcludeSniffs": [],
  "phpSniffer.standard": "phpcs.xml",
  "phpSniffer.windowsHardkill": false,
  "phpSniffer.windowsPhpCli": "php.exe"
}
```

**Additional context**
[vscode-php-sniffer-bug.tar.gz](https://github.com/wongjn/vscode-php-sniffer/files/3305633/vscode-php-sniffer-bug.tar.gz)

**Comment by wongjn** [comment]
The extension passes the content of the file to phpcs in `stdin` with its file path; in essence, the extension passes a single file to the command. With this in mind, the [documentation for the xml config file](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset) states that:

> […] specifying any file or directory path on the command line will ignore all file tags.

So as a work-around for now, you could try excluding the test file as such by adding the following to `phpcs.xml`:

```xml
<exclude-pattern>*/tests/*</exclude-pattern>
```

That said, I will look into adding `<file>` tag parsing in a future update.

**Comment by pattisahusiwa** [comment]
Yes, I've digged into the source code and tried to execute the same options directly in the console. `phpcs` just ignore the file.
